chatlas/_chat.py

```diff
@@ -188,6 +188,51 @@ def tokens(self) -> list[tuple[int, int] | None]:
         """
         return [turn.tokens for turn in self._turns]
 
+    def token_count(
+        self,
+        *args: Content | str,
+        extract_data: bool = False,
+    ) -> int:
+        """
+        Get the token count for the given input.
+
+        This is useful for estimating the number of tokens your input will cost
+        before sending it to the model.
```

Comment:
```suggestion
        Estimate the token size of input content. This can help determine whether input(s) 
        and/or conversation history (i.e., `.get_turns()`) should be reduced in size before
        sending it to the model.
```